routing/unified_edges.go

```diff
@@ -218,7 +218,10 @@ func (u *edgeUnifier) getEdgeLocal(amt lnwire.MilliSatoshi,
 		maxBandwidth = bandwidth
 
 		// Update best edge.
-		bestEdge = &unifiedEdge{policy: edge.policy}
+		bestEdge = &unifiedEdge{
+			policy:   edge.policy,
+			capacity: edge.capacity,
```

Comment:
we do not have the local balance here right?

Reply:
In principle, yes given by `bandwitdh`. I just the added capacity for consistency, it doesn't have an effect here, because local channel probability calculation is done differently. We assume that we can send over our own channels, unless the amount we try to send is below the bandwidth.